.github/buildomat/force-git-over-https.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# The token authentication mechanism that affords us access to other private
+# repositories requires that we use HTTPS URLs for GitHub, rather than SSH.
+#
+override_urls=(
+    'git://github.com/'
+    'git@github.com:'
+    'ssh://github.com/'
+    'ssh://git@github.com/'
+    'git+ssh://git@github.com/'
+)
+for (( i = 0; i < ${#override_urls[@]}; i++ )); do
+	git config --add --global url.https://github.com/.insteadOf \
```

Comment:
Dumb question: why don't we need quotes around "https://github.com", i.e.

```
git config --add --global url."https://github.com/".insteadOf
```
?

Reply:
I uh borrowed this setup from omicron https://github.com/oxidecomputer/omicron/blob/main/tools/include/force-git-over-https.sh